uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/request.controller.spec.js

```diff
@@ -376,7 +376,7 @@
 
                 leaveRequest.contact_id = CRM.vars.leaveAndAbsences.contactId.toString();
 
-                initTestController({ isSelfRecord: true, leaveRequest: leaveRequest });
+                initTestController({ leaveRequest: leaveRequest, isSelfRecord: true });
```

Comment:
why did you remove `isSelfRecord: true` from other tests but left this one?

Reply:
The implementation of `isSelfRecord` was changed and now is used only in one place in the code:

```js
function changeStatusBeforeSave () {
  if (vm.isSelfRecord) {
    vm.request.status_id = vm.requestStatuses[sharedSettings.statusNames.awaitingApproval].value;
  } else if (vm.canManage) {
    vm.request.status_id = vm.newStatusOnSave || vm.request.status_id;
  }
}
```
and exactly this place is tested: 

```js
it('status changes to waiting approval before calling API', function () {
```